packages/did/src/Did.chain.ts

```diff
@@ -418,13 +419,18 @@ export async function getStoreTx(
     .createType(api.tx.did.create.meta.args[0].type.toString(), apiInput)
     .toU8a()
 
+  // signers can identify using the authentication public key encoded as address or hex, or a light did authentication key
```

Comment:
We do have other functions to match a DID, also allowing a fragment to be present. Would that make more sense? If there's any match, check that the fragment is `#authentication`.

Reply:
true, I'll check that

Reply:
2f7fe68d90fa644c87eb498001bc5b8ae23e99c8